akka-persistence/src/main/scala/akka/persistence/fsm/PersistentFSM.scala

```diff
@@ -54,6 +54,16 @@ trait PersistentFSM[S <: FSMState, D, E] extends PersistentActor with Persistent
   private var currentStateTimeout: Option[FiniteDuration] = None
 
   /**
+   * "akka.persistence.fsm.enable-snapshot-after"
+   */
+  private val enableSnapshotAfter = context.system.settings.config.getBoolean("akka.persistence.fsm.enable-snapshot-after")
+
+  /**
+   * "akka.persistence.fsm.snapshot-after"
+   */
+  private val snapshotAfter = context.system.settings.config.getInt("akka.persistence.fsm.snapshot-after")
```

Comment:
Argh.... this was **NOT** ok.

I did the following test and it failed :(

- my library 1.0.0 compiled with Akka 2.5.0
- compiled Akka including my change, as 2.5.1
- An app 1.0.0 compiled with Akka 2.5.1 and my library 1.0.0

Then I got the following exception:

```
Uncaught error from thread [MyPFSMApp-akka.actor.default-dispatcher-2] shutting down JVM since 'akka.jvm-exit-on-fatal-error' is enabled for ActorSystem[MyPFSMApp]
java.lang.AbstractMethodError: com.mypkg.MyPFSM.akka$persistence$fsm$PersistentFSM$_setter_$akka$persistence$fsm$PersistentFSM$$snapshotAfter_$eq(Lscala/Option;)V
        at akka.persistence.fsm.PersistentFSM$class.$init$(PersistentFSM.scala:59)
        at com.mypkg.MyPFSM.<init>(MyPFSM.scala:15)
        at com.mypkg.MyPFSMApp$$anonfun$1.apply(MyPFSMApp.scala:13)
        at com.mypkg.MyPFSMApp$$anonfun$1.apply(MyPFSMApp.scala:13)
        at akka.actor.TypedCreatorFunctionConsumer.produce(IndirectActorProducer.scala:87)
        at akka.actor.Props.newActor(Props.scala:213)
        at akka.actor.ActorCell.newActor(ActorCell.scala:555)
        at akka.actor.ActorCell.create(ActorCell.scala:581)
        at akka.actor.ActorCell.invokeAll$1(ActorCell.scala:454)
        at akka.actor.ActorCell.systemInvoke(ActorCell.scala:476)
        at akka.dispatch.Mailbox.processAllSystemMessages(Mailbox.scala:282)
        at akka.dispatch.Mailbox.run(Mailbox.scala:223)
        at akka.dispatch.Mailbox.exec(Mailbox.scala:234)
        at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
        at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
        at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
        at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```